cognite/client/data_classes/relationships.py

```diff
@@ -258,5 +408,13 @@ def _get_update_properties(cls) -> list[PropertySpec]:
         ]
 
 
-class RelationshipList(CogniteResourceList[Relationship]):
+class RelationshipList(CogniteResourceList[Relationship], ExternalIDTransformerMixin):
     _RESOURCE = Relationship
+
+    def as_write(self) -> RelationshipWriteList:
+        """Returns this RelationshipList in its writing version."""
+        return RelationshipWriteList([item.as_write() for item in self.data], cognite_client=self._cognite_client)
```

Comment:
Should we abstract this function to the base class?

Reply:
Advantage of having it as is the docstrings becomes specific.